### ISSUE TITLE
Compatibility with webpack 3

### ISSUE THREAD

**Issue by Stefku** [lettertwo/appcache-webpack-plugin#21]
When used with webpack 3.0.0 npm reported a warning

```
npm WARN appcache-webpack-plugin@1.3.0 requires a peer of webpack@>=0.11 <2.0.0 || >=2.1.0-beta.13 <3.0.0 but none was installed.
```

**Comment by PierBover** [comment]
But does it work?

Edit: yes it does.

**Comment by Stefku** [comment]
Yes it does.

**Comment by lettertwo** [comment]
This warning *should* be fixed as of [v1.4.0](https://www.npmjs.com/package/appcache-webpack-plugin/v/1.4.0).
